src/Menu/Menu.js

```diff
@@ -83,10 +83,11 @@ export default class Menu extends Component {
   menuList = undefined;
 
   handleEnter = (element) => {
-    const list = findDOMNode(this.menuList);
+    const list = reactDomUtil.findDOMNode(this.menuList);
 
     if (this.menuList && this.menuList.selectedItem) {
-      findDOMNode(this.menuList.selectedItem).focus(); // eslint-disable-line react/no-find-dom-node
+      reactDomUtil.findDOMNode(this.menuList.selectedItem)
+                                            .focus(); // eslint-disable-line react/no-find-dom-node
```

Comment:
> I can move the indent to be just one tab?

That's the convention we are using. If we were using https://github.com/prettier/prettier, that would be handled automatically.